File: java/BillsApi/src/main/java/com/apilangarena/javabillsapi/presentation/dto/BillResponse.java
package com.apilangarena.javabillsapi.presentation.dto;

import java.math.BigDecimal;
import java.time.LocalDate;

public record BillResponse(
    Long id,
    String billNumber,
    LocalDate issuedAt,
    BigDecimal total,
    String currency
) {
}

File: kotlin/BillsApi/src/main/kotlin/com/apilangarena/billsapi/presentation/dto/BillResponse.kt
package com.apilangarena.billsapi.presentation.dto

import java.math.BigDecimal
import java.time.LocalDate

data class BillResponse(
    val id: Long,
    val billNumber: String,
    val issuedAt: LocalDate,
    val total: BigDecimal,
    val currency: String,
)
